pytorch_lightning/callbacks/__init__.py

```diff
@@ -11,7 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-from pytorch_lightning.callbacks.base import Callback
+from pytorch_lightning.callbacks.callback import Callback
```

Comment:
Please keep around the original file at `callbacks/base.py` with a deprecation warning that users should import from `callbacks/callback.py` instead.

Otherwise, this risks being a breaking change for users who currently depend on `pytorch_lightning.callbacks.base.Callback`